src/web/components/elements/CommentBlockComponent.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { css } from 'emotion';
+import { unescapeData } from '@root/src/lib/escapeData';
+import { textSans } from '@guardian/src-foundations/typography';
+import { palette, space } from '@guardian/src-foundations';
+
+type Props = {
+    body: string;
+    avatarURL: string;
+    profileURL: string;
+    profileName: string;
+    dateTime: string;
+};
+
+const commentWrapper = css`
+    ${textSans.medium()};
+    padding: 4px 0 0 10px;
+    border-left: 1px ${palette.neutral[86]} solid;
+    border-top: 1px ${palette.neutral[86]} solid;
+    margin-top: 12px;
+`;
+
+const bodyContent = css`
+    p {
+        word-break: break-word;
+        margin-bottom: 8px;
+    }
+`;
+
+const profileWrapper = css`
+    display: flex;
+    margin-top: ${space[4]}px;
+    margin-bottom: ${space[12]}px;
+`;
+
+const avatar = css`
```

Comment:
as a conversion we try to call these variables with the suffix `-Styles` so here we could call it `avatarStyles`